e2e-tests/themes/development-runtime/cypress/support/index.js

```diff
@@ -1 +1,3 @@
 import "gatsby-cypress"
+import "@testing-library/cypress/add-commands"
+import "./commands"
```

Comment:
This file is missing newline at end of file (your editor can probably do this automatically..)
(warning icon wont show up in discussion tab)

Reply:
Ya, I saw it. I'm not sure why our setup isn't catching that in pre-commit or even in warnings in the editor.